app/src/main/res/values/colors.xml

```diff
@@ -4,6 +4,7 @@
     <color name="colorSecondary">#F2B872</color>
     <color name="colorAccent">#52D9CB</color>
     <color name="grey">#F2F2F2</color>
-    <color name="black">#0D0126</color>
```

Comment:
The black color is not a plain black but rather a shade of blue that provide contrast, but if you want to change it why not 🤷‍♂️.

Reply:
The result was not really pretty with the previous black, but as it is just a constant we can change it whenever we want :)